projects/media-viewer/src/lib/media-viewer/annotation/annotation-api-http.service.ts

```diff
@@ -4,19 +4,18 @@ import { tap } from 'rxjs/operators';
 import { isPlatformServer } from '@angular/common';
 import {HttpResponse, HttpClient} from '@angular/common/http';
 import {Injectable, PLATFORM_ID, Inject} from '@angular/core';
-import {IDocumentTask} from './document-task.model';
 import {Annotation, IAnnotation, IAnnotationSet} from './annotation-set.model';
-import { EmLoggerService } from '../logging/em-logger.service';
+import { EmLoggerService } from '../../logging/em-logger.service';
 
 @Injectable()
-export class ApiHttpService {
+export class AnnotationApiHttpService {
     private baseUrl: string;
 
     constructor(private log: EmLoggerService,
                 private httpClient: HttpClient,
                 @Inject(PLATFORM_ID) private platformId,
                 private transferState: TransferState) {
-        log.setClass('ApiHttpService');
+        log.setClass('AnnotationApiHttpService');
```

Comment:
`log.setClass(this.constructor.name)` would probably do the trick in a more generic way. Though it seems for a really odd thing to be doing in the first place. Might dig into that later